joomla-extensions/weblinks
# Changed language strings so that the parameter "image float" in the config is more concrete

Pull Request for Issue https://github.com/joomla-extensions/weblinks/issues/360.

### Summary of Changes
I have made the language strings more accurate to the float option for the images and inserted dividing lines. 
The dividing lines should make it clearer that the float values are not part of the icons.
I use this new language strings in the config.xml (param) and in the weblink.xml (option of the weblink). So we do not need the general language string any more.
(By the way I added the option showon="save_history:1", so that the version limit field is only shown if versions are active.)


### Testing Instructions
Open the global configuration for web links (JOOMLA/administrator/index.php?option=com_config&view=component&component=com_weblinks) and see the new language strings.
After that create a new web link and see the strings in the images tab.
